common/app/shoppingcart/PaymentInfoBean.java

```diff
@@ -0,0 +1,37 @@
+package shoppingcart;
+
+import common.contexts.UserContext;
+import io.sphere.sdk.carts.PaymentInfo;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+
+import javax.annotation.Nullable;
+
+public class PaymentInfoBean extends Base {
+
+    private String type;
+
+    public PaymentInfoBean() {
+    }
+
+    public PaymentInfoBean(final @Nullable PaymentInfo paymentInfo, final UserContext userContext) {
+        if (paymentInfo != null) {
+            this.type = paymentInfo.getPayments().stream()
+                    .filter(ref -> ref.getObj() != null)
+                    .map(ref -> {
+                        final PaymentMethodInfo methodInfo = ref.getObj().getPaymentMethodInfo();
+                        return methodInfo.getName().find(userContext.locales()).orElse(methodInfo.getMethod());
+                    })
+                    .findFirst() // TODO Handle multiple payments
```

Comment:
Actually I put in TODO all those issues that cannot be handled by the app alone, so in this case it needs support from the theme. I need to recollect all those and create the corresponding issues in theme.